models/topics.models.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection")
+
+fetchTopics = () => {
+  return db
+  .query('SELECT * FROM topics').then((result) => {
+    return result.rows;
+  })
+  .catch((err) => {
```

Comment:
you don't need your catch in the model if you put it at then end of the promise chain in the controller it will handle any errors